test/xla_dist_test.py

```diff
@@ -237,15 +233,13 @@ def list_instances_fn(*args, **kwargs):
   return compute_service
 
 
-def build_mock_services_fn(mock_compute_service, mock_tpu_service):
+def build_mock_services_fn(mock_compute_service):
 
   def mock_google_services(serviceName, version, **kwargs):
     if serviceName == 'compute':
       return mock_compute_service
-    elif serviceName == 'tpu':
-      return mock_tpu_service
     else:
-      raise RuntimeError('Service name "{}" is not mocked.'.format(serviceName))
+      raise RuntimeError(f'Service name "{serviceName}" is not mocked.')
```

Comment:
In general, do not mix style changes with real changes.

Reply:
Ack. I've at least undone the style change in `xla_dist.py` but will keep the style changes in the unit tests. Thanks will keep in mind.